docs/contrib/build.rst

```diff
@@ -111,75 +114,62 @@ The following shows how to set ``nativeGC`` on all the projects.
 
 The same process above will work for setting `nativeMode`.
 
-Detailed build information
---------------------------------
-Knowledge of the Scala Native build system is very helpful for many tasks.
-In general the code is built and published to your local Ivy repository
-using the `sbt` `publishLocal` command so that other components in the
-system can depend on each other via normal `sbt` dependencies. Although the
-``build.sbt`` file and other code in the system is the way to learn the system
-thoroughly, the following sections will give information that should be helpful
-to get started.
-
-The build has roughly four groups of sub-projects as follows:
+Organization of the build
+-------------------------
+The build has roughly five groups of sub-projects as follows:
 
-1.  The Native Scala Compiler plugin and libraries. This first list is in
-    reverse dependency order.
+1.  The compiler plugin, which generates NIR files. It is used in all the
+    Scana Native artifacts in the build, with
+    ``.dependsOn(nscplugin % "plugin")``. This is a JVM project.
 
-    When we do a `rebuild` or `dirty-rebuild` the alias calls
-    `scalalib/publishLocal` first. The `scalalib` project depends on all the
-    projects above it in the following list. This means that the `nscplugin`
-    must be built and published locally first. The `nscplugin` project contains
-    the Scala compiler plugin which is normal Scala (JVM). The plugin compiles
-    Scala code into NIR which is needed for all the Scala Native libraries
-    lower in the list below.
+    - ``nscplugin``
 
-    - `nscplugin`
+2.  The Scala Native core libraries. Those are core artifacts which the sbt
+    plugin adds to the ``Compile`` configuration of all Scala Native projects.
+    The libraries in this group are themselves Scala Native projects.
 
-    - `nativelib`
+    - ``nativelib``
 
-    - `clib`
+    - ``clib``
 
-    - `posixlib`
+    - ``posixlib``
 
-    - `javalib`
+    - ``javalib``
 
-    - `auxlib`
+    - ``auxlib``
 
-    - `scalalib`
+    - ``scalalib``
```

Comment:
Since these are not in alphabetical order, or in another obvious order,
it may be worthwhile to say that projects later in the list depend on those
earlier in the list.

Reply:
Done.